src/applications/hca/helpers.jsx

```diff
@@ -14,9 +14,40 @@ import {
 } from 'platform/forms-system/src/js/helpers';
 import { getInactivePages } from 'platform/forms/helpers';
 import { isValidDate } from 'platform/forms/validations';
+import { isInMVI } from 'platform/user/selectors';
 
 import facilityLocator from '../facility-locator/manifest';
 
+export function prefillTransformer(pages, formData, metadata, state) {
```

Comment:
Definitely seems like a better way to set the view field. With this, I think you can remove the obsolete code from `IDPage`.

Reply:
thanks for catching that. I might need to rework the redirection logic on the `IDPage` now...